<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UcPrintLabel</class>
 <widget class="QWidget" name="UcPrintLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbSave">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/katynko/svg/document-save.svg</normaloff>:/katynko/svg/document-save.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/katynko/svg/dialog-cancel.svg</normaloff>:/katynko/svg/dialog-cancel.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Ignored</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbSettings">
           <property name="text">
            <string>Print settings</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/katynko/svg/applications-system.svg</normaloff>:/katynko/svg/applications-system.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbHtml">
           <property name="text">
            <string>Form settings</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/katynko/svg3/run-build.svg</normaloff>:/katynko/svg3/run-build.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>EUI64</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Model</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="4">
            <widget class="QRadioButton" name="rbroomtemp">
             <property name="text">
              <string>0C to 65C</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>MAC</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lemac"/>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>IMEI</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>S/N</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QComboBox" name="cbxModel"/>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lesn"/>
           </item>
           <item row="7" column="0" colspan="4">
            <widget class="QRadioButton" name="rbwidetemp">
             <property name="text">
              <string>-40C to +85C</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="leeui64"/>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="leimei"/>
           </item>
           <item row="9" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Part number</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2" colspan="2">
            <widget class="QLineEdit" name="lePartNumber"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="vl4previewMain"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
